### PR TITLE
Fix header dropdown styles

### PR DESCRIPTION
**Pull request by brionmario**
### Purpose
1. Add pointing direction to user dropdown & wrap with Menu.Item.
2. Refactor `Header` component. 
3. Fix product logo alignment issue
4. Update Console & My Account icons
5. Introduce inverted option to GenericIcon
6. Add global inverted variables
7. Remove app switch card

### Related Issues
- https://github.com/wso2-enterprise/asgardeo-product/issues/3304
### Checklist
- [x] e2e cypress tests locally verified.
- [x] Manual test round performed and verified.
- [ ] UX/UI review done on final implementation
- [ ] Documentation provided. (Add links)
- [ ] Unit tests provided. (Add links if any)
- [ ] Integration tests provided. (Add links if any)

### Related PRs
- https://github.com/wso2-enterprise/asgardeo-app-extensions/pull/607

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [x] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

File: modules/react-components/src/components/header/header.tsx [hunk 1]
```diff
@@ -211,274 +217,274 @@ export const Header: FunctionComponent<HeaderPropsInterface> = (
...
-          </Container>
-          { children }
-      </Menu>
+                                                            return (
+                                                                <Dropdown.Item
+                                                                    key={ index }
+                                                                    className="action-panel"
+                                                                    onClick={ onClick }
+                                                                    data-testid={
+                                                                        `${ testId }-dropdown-link-${
```
Comment (inthirakumaaran) [inline modules/react-components/src/components/header/header.tsx:458]: Do we need to enforce the replacement of  " " to "-" here?

File: modules/react-components/src/components/dropdown/app-switcher/app-switcher.tsx [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+     */
+    triggerIcon?: ReactNode;
+}
+
+export interface AppSwitcherItemPropsInterface extends TestableComponentInterface, DropdownItemProps {
+
+    /**
+     * Short description about the app.
+     */
+    description: string;
```
Comment (inthirakumaaran) [inline modules/react-components/src/components/dropdown/app-switcher/app-switcher.tsx:59]: Can we make the description optional, WDYT?

File: modules/react-components/src/components/header/header.tsx (hunk 1 above)
Comment (brionmario) [reply modules/react-components/src/components/header/header.tsx:458]: yes. The `name` here can contain `spaces`. This block was only reformatted in this PR.

File: modules/react-components/src/components/dropdown/app-switcher/app-switcher.tsx (hunk 2 above)
Comment (brionmario) [reply modules/react-components/src/components/dropdown/app-switcher/app-switcher.tsx:59]: We can't maintain consistent UI if we make this optional. Some apps will have a description and some will not, which might result in empty spaces in the dropdown.

File: apps/console/src/features/core/components/header.tsx [hunk 3]
```diff
@@ -126,6 +129,47 @@ export const Header: FunctionComponent<HeaderPropsInterface> = (
...
+                tooltip={ t("console:common.header.appSwitch.tooltip") }
+                apps={ [
+                    {
+                        "data-testid": "app-switch-console",
+                        description: t("console:common.header.appSwitch.console.description"),
+                        enabled: true,
+                        icon: AppSwitcherIcons().console,
+                        name: t("console:common.header.appSwitch.console.name"),
+                        onClick: () => {
+                            window.open(accountAppURL,"_self");
```
Comment (inthirakumaaran) [inline apps/console/src/features/core/components/header.tsx:155]: Is the URL correct?

**Review by inthirakumaaran — APPROVED** [review]

File: apps/console/src/features/core/components/header.tsx (hunk 3 above)
Comment (brionmario) [reply apps/console/src/features/core/components/header.tsx:155]: Yes. Target is set to `_self`.

**Pull request merged**